docs/index.md

```diff
@@ -80,12 +80,15 @@ protractorImageComparison
 | options.hideScrollBars | <code>boolean</code> | Hide all scrolls on a page (default:true) |
 | options.nativeWebScreenshot | <code>boolean</code> | If a native screenshot of a device (complete screenshot) needs to be taken (default:false) |
 | options.blockOutStatusBar | <code>boolean</code> | If the statusbar on mobile / tablet needs to blocked out by default |
-| options.ignoreAntialiasing | <code>boolean</code> | compare images an discard anti aliasing |
+| options.ignoreAlpha | <code>boolean</code> | compare images and discard alpha |
+| options.ignoreAntialiasing | <code>boolean</code> | compare images and discard anti aliasing |
 | options.ignoreColors | <code>boolean</code> | Even though the images are in colour, the comparison wil compare 2 black/white images |
+| options.ignoreLess | <code>boolean</code> | compare images and compare with less (red = 16, green = 16, blue = 16, alpha = 16, minBrightness=16, maxBrightness=240) |
+| options.ignoreNothing | <code>boolean</code> | compare images and compare with less (red = 0, green = 0, blue = 0, alpha = 0, minBrightness=0, maxBrightness=255) |
```

Comment:
Maybe it would be better to say 'compare images and compare with nothing'?

Reply:
Also, please update it in other places. I can mark them if needed. If I can help you with this update somehow, please notify me)